x/auth/ante/ante_test.go

```diff
@@ -347,40 +305,20 @@ func TestAnteHandlerAccountNumbersAtBlockHeightZero(t *testing.T) {
 			false,
 			sdkerrors.ErrUnauthorized,
 		},
-		{
-			"new tx with another signer and incorrect account numbers",
```

Comment:
To minimize these changes, you may want to a a "skip" filed in struct `TestCase`, and use it to disable test cases (by calling `t.Skip("not relevant to Heimdall")` at the right place).

Reply:
Good one, will do, thanks

Reply:
Done in latest commit.

Reply:
Resolved